.appveyor.yml

```diff
@@ -69,3 +69,11 @@ for:
       - cd build
       - cmake .. -G "Ninja" -DCMAKE_BUILD_TYPE:STRING="%configuration%" -DCMAKE_C_COMPILER:FILEPATH="cl.exe" -DCMAKE_CXX_COMPILER:FILEPATH="cl.exe"
       - cmake --build . -j 4
+
+after_build:
+    - 7z a mxBuild.zip C:\projects\mxengine\samples\*
+    - XCOPY mxBuild.zip C:\projects\mxengine\
```

Comment:
xcopy is windows specific command. We have both Linux and Windows. Also use '/' instead of '\'